src/EventSubscriber/DataSubscriber.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\EventSubscriber;
+
+use Platformsh\DevRelBIPhpSdk\Symfony\EventSubscriber\DataEventSubscriber;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+
+class DataSubscriber extends DataEventSubscriber
+{
+    protected function logEvents(RequestEvent $event): void
+    {
+        $isLoggedIn = $this->isLoggedIn();
+        $request = $event->getRequest();
+        $path = $request->getPathInfo();
+
+        switch (true) {
+            case '/landing' === $path:
```

Comment:
this means that a malicious user could find the final step by watching at this file :P

Reply:
we should probably use environment variable for this, to hide it from the final user